cmd/prometheus/main.go

```diff
@@ -35,14 +35,14 @@ import (
 
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
-	conntrack "github.com/mwitkow/go-conntrack"
```

Comment:
These aliases help in knowing what name to be use to refer the package (same goes for kingpin and yaml). Because those package names are not clear by just looking at the import path.

Reply:
Oh, I understand it.
Thank you.

Reply:
The Yoda station need to update in the document tsdb/index/postings_test.go.

Reply:
？？